doc/sphinx/conf.py

```diff
@@ -27,7 +27,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["sphinx.ext.autosectionlabel", "sphinx_multiversion"]
```

Comment:
I removed the sectionautolabel extension as it resulted in quite a mess (with duplicated labels and inconsistent referencing). Based on what I've read in the wake of that mess is that quite often people recommend to be specific about labels, i.e., add one where it's needed (similar to latex) rather than rely on headings (that may change, and thus break links).
So for links within the same file, we can continue to use `could be a section title`_ but across files, we should use the explicit :ref:`my label` notation.

Reply:
Ok that makes sense to me.